app/assets/stylesheets/modules/star.scss

```diff
@@ -0,0 +1,18 @@
+.fa-paw {
+  font-size: 1em;
+  color: gold;
```

Comment:
Color `gold` should be written in hexadecimal form as `#ffd700`<br>Color literals like `gold` should only be used in variable declarations; they should be referred to via variable everywhere else.